packages/common/editors/__tests__/EditorContainer.spec.js

```diff
@@ -172,13 +172,17 @@ describe('Editor Container Tests', () => {
 
     it('should not commit if any element inside the editor is clicked', () => {
       const editor = wrapper.find(TestEditor);
-      editor.simulate('click');
+      editor.simulate('mousedown');
 
       expect(props.onCommit.calls.count()).toEqual(0);
     });
 
     it('should commit if any element outside the editor is clicked', () => {
-      document.querySelector('.container').click();
+      const clickEvent = document.createEvent('MouseEvents');
+      clickEvent.initEvent('mousedown', true, true);
+      const input = document.createElement('input');
+      document.querySelector('.container').appendChild(input);
+      input.dispatchEvent(clickEvent);
```

Comment:
Do we need to dispatch the event on an input? We could just mousedown on the body.

Reply:
It works. Previously I was dispatching an event on the container and it was returning a [false positive](https://github.com/adazzle/react-data-grid/pull/1404/files#diff-a4a272b1d00cac994690178e6a7ca36aR180) so I decided to use an explicit sibling node.